lib/ui/barcode_preview_multi_image.dart

```diff
@@ -0,0 +1,89 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:scanbot_sdk/barcode_scanning_data.dart';
+import 'package:scanbot_sdk_example_flutter/ui/pages_widget.dart';
+
+import '../main.dart';
+import 'barcode_preview.dart';
+
+class MultiImageBarcodesResultPreviewWidget extends StatelessWidget {
+  final List<BarcodeScanningResult> lstPreviewItems;
```

Comment:
Consider using full word instead of an abbreviation

Reply:
Fixed 👍